<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VectorWidget</class>
 <widget class="QWidget" name="VectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widgetLow" native="true">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="addLowButton">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/main.qrc">
          <normaloff>:/new/prefix1/plus.png</normaloff>:/new/prefix1/plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeLowButton">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/main.qrc">
          <normaloff>:/new/prefix1/minus.png</normaloff>:/new/prefix1/minus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widgetSide" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="addSideButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/main.qrc">
          <normaloff>:/new/prefix1/plus.png</normaloff>:/new/prefix1/plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeSideButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/main.qrc">
          <normaloff>:/new/prefix1/minus.png</normaloff>:/new/prefix1/minus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>249</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addLowButton</sender>
   <signal>released()</signal>
   <receiver>VectorWidget</receiver>
   <slot>addEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeLowButton</sender>
   <signal>released()</signal>
   <receiver>VectorWidget</receiver>
   <slot>removeEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeSideButton</sender>
   <signal>released()</signal>
   <receiver>VectorWidget</receiver>
   <slot>removeEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addSideButton</sender>
   <signal>released()</signal>
   <receiver>VectorWidget</receiver>
   <slot>addEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addEntry()</slot>
  <slot>removeEntry()</slot>
 </slots>
</ui>
